cirq/google/engine/client/quantum_v1alpha1/types.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
```

Comment:
We ran into issues with this file. I don't remember if it was actually running it, or mypy, or tests but that's why I deleted it and created the types/__init__.py. If this works now, let's delete the types/__init__.py otherwise let's delete this and add duration_pb2 to types/__init__.py.

Reply:
Yes, it looks like mypy can't handle this file since it is dynamically adding items to the namespace.  I have removed it and added duration_pb2 to the types/__init__.py